Essentials/src/com/earth2me/essentials/RandomTeleport.java

```diff
@@ -155,12 +155,12 @@ public void cacheRandomLocations(Location center, double minRange, double maxRan
     // Calculates a random location asynchronously.
     private CompletableFuture<Location> calculateRandomLocation(Location center, double minRange, double maxRange) {
         CompletableFuture<Location> future = new CompletableFuture<>();
-        final int dx = RANDOM.nextBoolean() ? 1 : -1, dz = RANDOM.nextBoolean() ? 1 : -1;
+        double[] offset = getRandXySquare(minRange, maxRange);
```

Comment:
```suggestion
        double[] offsets = getOffsets(minRange, maxRange);
```
Might be a better name?